.circleci/config.yml

```
@@ -40,6 +40,13 @@ jobs:
             name: Checkout VSIP UI Tests
             command:
               git clone https://github.com/ministryofjustice/hmpps-vsip-ui-tests.git
+      - run:
+          name: Change to project directory
+          command: pwd && ls && cd /hmpps-vsip-ui-tests
```

```suggestion
          command: hmpps-vsip-ui-tests
```
I think, looking at the output of the last run, that the leading `/` is the problem - this folder isn't at the file system root